docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc

```diff
@@ -2,8 +2,56 @@
 = Adding projects to a devfile
 
 [role="_abstract"]
-This section describes how to add one or more projects to a devfile.
-
+This section describes how to add one or more projects to a devfile. Each starter project can contain either a `git` or `zip` object. See the following tables for project properties in a devfile:
```

Comment:
I just notice that we are switching context on talking about starterProjects and projects on this page. Title say sprojects but then we describe starterProjects and then having projects as examples. We probably need two different pages, one for starterProjects and one for projects since the allowed fields are different.

This may be a bigger change and  I am fine with addressing this in a different PR. To minimize the change, maybe use this project for project and create a different page for starterProjects (since they have different fields).

Reply:
Good catch, Elson. Yes, I like the idea of splitting it into two pages. And making those changes in a separate PR, especially since we're already saving work for these very docs for a future PR (like Tomas' feedback on expanding the descriptions). 

For the time being, did you want to leave this doc as is, or did you want me to remove the references to starterProjects?

Reply:
I think it is fine to leave it as is and fix that in a different PR.